### DISCUSSION TITLE
How to change height for history command?

### DISCUSSION THREAD

**Discussion by rohitkrishna094** [PatrickF1/fzf.fish#224] [Q&A] [answered]
When I run `ctrl+r` I get a fuzzy finder search which searches on my command history. However it only takes up half of the terminal height. Is there a way to increase this height so it takes up full terminal height?

I tried adding this in my fish config, but its not working:

```
set -x fzf_history_opts '--height=100%'
```

**Comment by PatrickF1** [comment, marked as answer]
That works for me! Please provide more information about your set up / try to debug it yourself for a little longer.

**Reply by rohitkrishna094** [reply]
Weird. For some reason if I use `FZF_DEFAULT_OPTS ` it works, but not `fzf_history_opts`

```fish
set -x FZF_DEFAULT_OPTS '--height=10%'
```

**Reply by PatrickF1** [reply]
Can you go through the checklist here https://github.com/PatrickF1/fzf.fish/issues/new?assignees=&labels=&template=bug_report.md&title=?
